ui/selectors/selectors.js

```diff
@@ -1008,6 +1008,9 @@ function getAllowedAnnouncementIds(state) {
     19: false,
     20: currentKeyringIsLedger && isFirefox,
     21: isSwapsChain,
+    ///: BEGIN:ONLY_INCLUDE_IN(blockaid)
+    23: true,
```

Comment:
shouldn't it be 22?

Reply:
There's already a 22 in the extension

```
shared/notifications/index.js
test/e2e/fixture-builder.js
```

But for some reason it wasn't added here.